src/affexpr.jl

```diff
@@ -146,8 +146,6 @@ product of two variables.
 """
 function add_to_expression! end
 
-# TODO: add deprecations for Base.push! and Base.append!
```

Comment:
This was a recent comment, shouldn't be removed yet.

Reply:
Fixed.